Demo/Demo/ViewModels/BaseViewModel.swift

```diff
@@ -102,14 +102,23 @@ class BaseViewModel: ObservableObject, PayPalWebCheckoutDelegate, CardDelegate {
 
     func checkoutWith(
         card: Card,
-        orderID: String
+        orderID: String,
+        shouldVault: Bool = false,
+        customerID: String = ""
     ) async {
         guard let config = await getCoreConfig() else {
             return
         }
         let cardClient = CardClient(config: config)
         cardClient.delegate = self
-        let cardRequest = CardRequest(orderID: orderID, card: card, sca: .scaAlways)
+        
+        var passedCustomerID: String?
+        if customerID.isEmpty {
+            passedCustomerID = nil
+        } else {
+            passedCustomerID = customerID
+        }
```

Comment:
```suggestion
        let passedCustomerID = customerID.isEmpty ? nil : customerID
```
🧶 : this should be equivalent. It's a bit more concise and eliminates the `var`.

Reply:
Addressed in commit 6fd6e1bb5e4234f8e0fc2d77a3ca77670eac6ee3